ethpm/package.py

```diff
@@ -371,7 +374,9 @@ def deployments(self) -> Union["Deployments", Dict[None, None]]:
         return Deployments(deployments, all_contract_instances, self.w3)
 
     @to_dict
-    def _get_all_contract_instances(self, deployments):
+    def _get_all_contract_instances(
+        self, deployments: Dict[str, Any]
```

Comment:
Aren't deployments structured?  Can you define a `TypedDict` in place of the `Any`?